---
layout: default
---

# The List of Accepted Papers

* __*A Gene Profiling Deconvolution Approach to Estimating Immune Cell Composition from Complex Tissues*__, Wen‐Yu Kuo, Shu‐Hwa Chen, Sheng‐Yao Su, Wei‐ChunChung, Jen‐Ming Ho, Henry Horng‐Shing Lu and Chung‐Yen Lin
* __*A new method to measure the semantic similarity from query phenotypic abnormalities to diseases based on the Human Phenotype Ontology*__, Xiaofeng Gong, Jianping Jiang, Zhongqu Duan and Hui Lu
* __*Adjacent Y-ion Ratio Distributions and Its Application in Peptide Sequencing*__, Tiancong Wang and Bin Ma
* __*An Approach for N-linked Glycan Identification from MS/MS Spectra by Target-Decoy Strategy*__, Weiping Sun, Yi Liu and Kaizhong Zhang
* __*An Exact Algorithm for Sorting by Weighted Preserving Genome Rearrangements*__, Tom Hartmann, Matthias Bernt and Martin Middendorf
* __*An Integrated Reconciliation Framework for Domain, Gene, and Species Level Evolution*__, Lei Li and Mukul S. Bansal
* __*Annotating activation/inhibition relationships to protein-protein interactions using Gene Ontology relations*__, Soorin Yim, Hasun Yu, Dongjin Jang and Doheon Lee
* __*Automated Transition Analysis of Activated Gene Regulation during Diauxic Nutrient Shift in Escherichia Coli and Adipocyte Differentiation in Mouse Cells*__, Yoichi Takenaka, Kazuma Mikami, Shigeto Seno and Hideo Matsuda
* __*cmFSM: A Scalable CPU-MIC Coordinated Drug-Finding Tool by Frequent Subgraph Mining*__, Shunyun Yang, Quan Zou, Shaoliang Peng and Runxin Guo
* __*CNNH_PSS: Protein 8-class Secondary Structure Prediction by Convolutional Neural Network with Highway*__, Jiyun Zhou, Hongpeng Wang, Zhishan Zhao, Ruifeng Xu and Qin Lu
* __*Comparisons of gene coexpression network modules in breast cancer and ovarian cancer*__, Shuqin Zhang
* __*Composite mathematical modeling of calcium signaling behind neuronal cell death in Alzheimer's disease*__, Bobby Ranjan, Ket Hing Chong and Jie Zheng
* __*Comprehensive analysis of coding-lncRNA gene co-expression network uncovers conserved functional lncRNAs in zebrafish*__, Wen Chen, Xuan Zhang, Jing Li, Shulan Huang, Shuanglin Xiang, Xiang Hu and Changning Liu
* __*Computational analysis of the receptor binding specificity of novel influenza A/H7N9 viruses*__, Xinrui Zhou, Jie Zheng, Fransiskus Xaverius Ivan, Rui Yin, Shoba Ranganathan, Vincent T. K. Chow and Chee Keong Kwoh
* __*Computing the Diameter of the Space of Maximum Parsimony Reconciliations in the Duplication-Transfer-Loss Model*__, Jordan Haack, Eli Zupke, Andrew Ramirez, Yi‐Chieh Wu and Ran Libeskind‐Hadas
* __*Counting motifs in dynamic networks*__, Kingshuk Mukherjee, Md Mahmudul Hasan, Christina Boucher and Tamer Kahveci
* __*Discovery of Boolean Metabolic Networks: Integer Linear Programming Based Approach*__, Yushan Qiu, Hao Jiang, Xiaoqing Cheng and Wai‐Ki Ching
* __*Dynamics of the human gut phageome during the antibiotics treatment*__, Anna Górska, Silke Peter, Matthias Willmann, Ingo Autenrieth, Robert Schlaberg and Daniel Huson
* __*Efficient Algorithms for Polyploid Haplotype Phasing*__, Dan He, Subrata Saha, Richard Finkers and Laxmi Parid
* __*Estimation of Diffusion Constants from Single Molecular Measurement without Explicit Tracking*__, Shunsuke Teraguchi and Yutaro Kumagai
* __*GATC: A Genetic Algorithm for gene Tree Construction under the Duplication-Transfer- Loss model of evolution*__, Emmanuel Noutahi and Nadia El‐Mabrouk
* __*Genome Read In-Memory (GRIM) Filter: Fast Location Filtering in DNA Read Mapping using Emerging Memory Technologies*__, Jeremie Kim, Damla Senol, Hongyi Xin, Donghyuk Lee, Saugata Ghose, Mohammed Alser, Hasan Hassan, Oguz Ergin, Can Alkan and Onur Mutlu
* __*Genome-wide identification and comprehensive analysis of microRNAs and phased small interfering RNAs in watermelon*__, Li Liu, Shuchao Ren, Junqiang Guo, Qingyi Wang, Xiaotuo
Zhang, Peiran Liao, Shipeng Li, Ramanjulu Sunkar and Yun Zheng
* __*Hardness of Covering Alignment: Phase Transition in Post-Sequence Genomics*__, Romeo Rizzi, Massimo Cairo, Veli Mäkinen, Alexandru I. Tomescu and Daniel Valenzuela
* __*Hardware Acceleration of BWA-MEM Genomic Short Read Mapping with Longer Read Length*__, Ernst Houtgast, Vlad‐Mihai Sima, Koen Bertels and Zaid Al‐Ars
* __*Hierarchical Structural Component Analysis of microRNA-mRNA Integration Analysis*__, Yongkang Kim, Sungyoung Lee, Sungkyoung Choi and Taesung Park
* __*Identification of Condition Specific Cis- and Trans-acting Elements in Plant promoters under Various Endo- and Exogenous Stimuli*__, Chi‐Nga Chow, Yi‐Fan Chiang‐Hsieh, Chia‐Hung Chien,
Han‐Qin Zheng, Nai‐Yun Wu, Kuan‐Chieh Tseng, Ping‐Fu Hou and Wen‐Chi Chang
* __*Identification of Usual Interstitial Pneumonia Pattern Using RNA-Seq and Machine Learning: Challenges and Solutions*__, Yoonha Choi, Tiffany Ting Liu, Daniel Pankratz, Thomas
Colby, Neil Barth, David Lynch, Sean Walsh, Ganesh Raghu, Giulia Kennedy and Jing Huang
* __*Integrated microRNA and mRNA expression profile analysis of Tumor-Associated Macrophages after Exposure to Single-Dose Irradiation*__, Wei‐Hsiang Kung, Chi‐Lung Lee, Chi‐Dung Yang, Ching‐
Fang Yu, Men‐Yee Chiew, Fang‐Hsin Chen and Hsien‐Da Huang
* __*KDiamend: a package for detecting key drivers in a molecular ecological network of disease*__, Mengxuan Lyu, Jiaxing Chen, Yiqi Jiang, Wei Dong, Zhou Fang and Shuaicheng Li
* __*Matching Algorithms for Assigning Orthologs after Duplication Events*__, Guillaume Fertin, Falk Hüffner, Christian Komusiewicz and Manuel Sorge
* __*MEGADOCK-Web: an integrated database of high-throughput structure-based protein- protein interaction predictions*__, Takanori Hayashi, Yuri Matsuzaki, Keisuke Yanagisawa,  Masahito Ohue and Yutaka Akiyama
* __*ncRNA-disease association prediction based on sequence information and tripartite network*__, Takuya Mori, Hayliang Ngouv, Morihiro Hayashida, Tatsuya Akutsu and Jose Nacher
* __*NGS-based likelihood ratio for identifying contributors in two- and three-person DNA mixtures*__, Joshua Mun Wei Chan, Zicheng Zhao, Shuai Cheng Li and Yen Kaow Ng
* __*Optimization of memory use of fragment extension-based protein-ligand docking with an original fast minimum cost flow algorithm*__, Keisuke Yanagisawa, Shunta Komine, Rikuto Kubota, Masahito Ohue and Yutaka Akiyama
* __*Ouroboros Resembling Competitive Endoge-nous Loop (ORCEL) in circular RNAs revealed through transcriptome sequencing dataset analysis*__, Yu‐Chen Liu, Hsiao‐Chin Hong, Chi‐Dung Yang, Wei‐Hsiang Lee, Hsin‐Tzu Huang and Hsien‐Da Huang
* __*Parallel Computation of the Burrows-Wheeler Transform of Short Reads Using Prefix Parallelism*__, Kouichi Kimura and Asako Koike
* __*Pathway-based approach using hierarchical components of rare variants to analyze multiple phenotypes*__, Sungyoung Lee, Yongkang Kim, Sungkyoung Choi, Heungsun Hwang and Taesung Park
* __*Phylogenetic signal from rearrangements in 18 Anopheles species by joint scaffolding extant and ancestral genomes*__, Yoann Anselmetti, Wandrille Duchemin, Eric Tannier, Cedric Chauve and Severine Berard
* __*Predicting and Understanding Comprehensive Drug-Drug Interactions via Semi- Nonnegative Matrix Factorization*__, Yu Hui, Kui‐Tao Mao, Jian‐Yu Shi, Hua Huang, Zhi Chen, Kai Dong and Siu‐Ming Yiu
* __*Prediction of enhancer-promoter interactions via natural language processing*__, Wanwen Zeng, Mengmeng Wu, Rui Jiang
* __*Probing the functions of long non-coding RNAs by exploiting the topology of global association and interaction network*__, Lei Deng, Hongjie Wu and Jinpu Zhang
* __*Protein complex prediction by date hub removal*__, Iana Pyrogova and Limsoon Wong
* __*RaptorX-Angle: real-value and confidence prediction of protein backbone dihedral angles through a hybrid method of clustering and deep learning
*__, Yujuan Gao, Sheng Wang, Minghua Deng and Jinbo Xu
* __*Regulation of dual specificity phosphatases in breast cancer during initial treatment with Herceptin: A Boolean model analysis*__, Petronela Buiga, Ari Elson, Lydia Tabernero and Jean‐Marc Schwartz
* __*Resolution effects in reconstructing ancestral genomes*__, Chunfang Zheng, Yuji Jeong, Madisyn Turcotte and David Sankoff
* __*Resource Cut, a New Bounding Procedure to Algorithms for Enumerating Tree-like Chemical Graphs*__, Yuhei Nishiyama, Aleksandar Shurbevski, Hiroshi Nagamochi and Tatsuya Akutsu
* __*SpliceVec: distributed feature representations for splice junction prediction*__, Aparajita Dutta, Tushar Dubey, Kusum Kumari Singh and Ashish Anand
* __*Taming Asynchrony for Attractor Detection in Large Boolean Networks*__, Andrzej Mizera, Jun Pang, Hongyang Qu and Qixia Yuan
* __*Tensor decomposition/principal component analysis based unsupervised feature extraction applied to brain gene expression and methylation profiles of social insects with multiple castes*__, Y‐H. Taguchi
* __*The Aquatic Animals’ Transcriptome Resource for Comparative Functional Analysis*__, Chih‐Hung Chou, Sheng‐Da Hsu, Chia‐Yu Liu, Yu‐Hung Chen, Yu‐Chen Liu, Wei‐Yun Huang, Chung‐Der Hsiao and Hsien‐Da Huang
* __*Towards Pan-Genome Read Alignment to Improve Variation Calling*__, Daniel Valenzuela, Tuukka Norri, Niko Välimäki, Esa Pitkänen and Veli Mäkinen
* __*VAReporter: Variant reporter for cancer research of massive parallel sequencing*__, Po‐Jung Huang, Chi‐Ching Lee, Ling‐Ya Chiu, Kuo‐Yang Huang, Yuan‐Ming Yeh, Chia‐Yu Yang, Cheng‐Hsun Chiu and Petrus Tang
