---
layout: default
---

# The List of Accepted Papers

* Yujuan Gao, Sheng Wang, Minghua Deng and Jinbo Xu, *RaptorX-Angle: real-value and confidence prediction of protein backbone dihedral angles through a hybrid method of clustering and deep learning*
* Kouichi Kimura and Asako Koike, *Parallel Computation of the Burrows-Wheeler Transform of Short Reads Using Prefix Parallelism*
* Jordan Haack, Eli Zupke, Andrew Ramirez, Yi‐Chieh Wu and Ran Libeskind‐Hadas, *Computing the Diameter of the Space of Maximum Parsimony Reconciliations in the Duplication-Transfer-Loss Model*
* Romeo Rizzi, Massimo Cairo, Veli Mäkinen, Alexandru I. Tomescu and Daniel Valenzuela, *Hardness of Covering Alignment: Phase Transition in Post-Sequence Genomics*
* Yu Hui, Kui‐Tao Mao, Jian‐Yu Shi, Hua Huang, Zhi Chen, Kai Dong and Siu‐Ming Yiu, *Predicting and Understanding Comprehensive Drug-Drug Interactions via Semi- Nonnegative Matrix Factorization*
* Andrzej Mizera, Jun Pang, Hongyang Qu and Qixia Yuan, *Taming Asynchrony for Attractor Detection in Large Boolean Networks*
* Wanwen Zeng, Mengmeng Wu, Rui Jiang, *Prediction of enhancer-promoter interactions via natural language processing*
* Chunfang Zheng, Yuji Jeong, Madisyn Turcotte and David Sankoff, *Resolution effects in reconstructing ancestral genomes*
* Li Liu, Shuchao Ren, Junqiang Guo, Qingyi Wang, Xiaotuo
Zhang, Peiran Liao, Shipeng Li, Ramanjulu Sunkar and Yun Zheng, *Genome-wide identification and comprehensive analysis of microRNAs and phased small interfering RNAs in watermelon*
* Tiancong Wang and Bin Ma, *Adjacent Y-ion Ratio Distributions and Its Application in Peptide Sequencing*
* Tom Hartmann, Matthias Bernt and Martin Middendorf, *An Exact Algorithm for Sorting by Weighted Preserving Genome Rearrangements*
* Yongkang Kim, Sungyoung Lee, Sungkyoung Choi and Taesung Park, *Hierarchical Structural Component Analysis of microRNA-mRNA Integration Analysis*
* Lei Li and Mukul S. Bansal, *An Integrated Reconciliation Framework for Domain, Gene, and Species Level Evolution*
* Chi‐Nga Chow, Yi‐Fan Chiang‐Hsieh, Chia‐Hung Chien,
Han‐Qin Zheng, Nai‐Yun Wu, Kuan‐Chieh Tseng, Ping‐Fu Hou and Wen‐Chi Chang, *Identification of Condition Specific Cis- and Trans-acting Elements in Plant promoters under Various Endo- and Exogenous Stimuli*
* Takanori Hayashi, Yuri Matsuzaki, Keisuke Yanagisawa,  Masahito Ohue and Yutaka Akiyama, *MEGADOCK-Web: an integrated database of high-throughput structure-based protein- protein interaction predictions*
* Daniel Valenzuela, Tuukka Norri, Niko Välimäki, Esa Pitkänen and Veli Mäkinen, *Towards Pan-Genome Read Alignment to Improve Variation Calling*
* Sungyoung Lee, Yongkang Kim, Sungkyoung Choi, Heungsun Hwang and Taesung Park, *Pathway-based approach using hierarchical components of rare variants to analyze multiple phenotypes*
* Yoann Anselmetti, Wandrille Duchemin, Eric Tannier, Cedric Chauve and Severine Berard, *Phylogenetic signal from rearrangements in 18 Anopheles species by joint scaffolding extant and ancestral genomes*
* Xiaofeng Gong, Jianping Jiang, Zhongqu Duan and Hui Lu, *A new method to measure the semantic similarity from query phenotypic abnormalities to diseases based on the Human Phenotype Ontology*
* Chih‐Hung Chou, Sheng‐Da Hsu, Chia‐Yu Liu, Yu‐Hung Chen, Yu‐Chen Liu, Wei‐Yun Huang, Chung‐Der Hsiao and Hsien‐Da Huang, *The Aquatic Animals’ Transcriptome Resource for Comparative Functional Analysis*
* Dan He, Subrata Saha, Richard Finkers and Laxmi Parid, *Efficient Algorithms for Polyploid Haplotype Phasing*
* Jeremie Kim, Damla Senol, Hongyi Xin, Donghyuk Lee, Saugata Ghose, Mohammed Alser, Hasan Hassan, Oguz Ergin, Can Alkan and Onur Mutlu, *Genome Read In-Memory (GRIM) Filter: Fast Location Filtering in DNA Read Mapping using Emerging Memory Technologies*
* Yuhei Nishiyama, Aleksandar Shurbevski, Hiroshi Nagamochi and Tatsuya Akutsu, *Resource Cut, a New Bounding Procedure to Algorithms for Enumerating Tree-like Chemical Graphs*
* Joshua Mun Wei Chan, Zicheng Zhao, Shuai Cheng Li and Yen Kaow Ng, *NGS-based likelihood ratio for identifying contributors in two- and three-person DNA mixtures*
* Xinrui Zhou, Jie Zheng, Fransiskus Xaverius Ivan, Rui Yin, Shoba Ranganathan, Vincent T. K. Chow and Chee Keong Kwoh, *Computational analysis of the receptor binding specificity of novel influenza A/H7N9 viruses*
* Yushan Qiu, Hao Jiang, Xiaoqing Cheng and Wai‐Ki Ching, *Discovery of Boolean Metabolic Networks: Integer Linear Programming Based Approach*
* Wen Chen, Xuan Zhang, Jing Li, Shulan Huang, Shuanglin Xiang, Xiang Hu and Changning Liu, *Comprehensive analysis of coding-lncRNA gene co-expression network uncovers conserved functional lncRNAs in zebrafish*
* Lei Deng, Hongjie Wu and Jinpu Zhang, *Probing the functions of long non-coding RNAs by exploiting the topology of global association and interaction network*
* Emmanuel Noutahi and Nadia El‐Mabrouk, *GATC: A Genetic Algorithm for gene Tree Construction under the Duplication-Transfer- Loss model of evolution*
* Jiyun Zhou, Hongpeng Wang, Zhishan Zhao, Ruifeng Xu and Qin Lu, *CNNH_PSS: Protein 8-class Secondary Structure Prediction by Convolutional Neural Network with Highway*
* Yoonha Choi, Tiffany Ting Liu, Daniel Pankratz, Thomas
Colby, Neil Barth, David Lynch, Sean Walsh, Ganesh Raghu, Giulia Kennedy and Jing Huang, *Identification of Usual Interstitial Pneumonia Pattern Using RNA-Seq and Machine Learning: Challenges and Solutions*
* Keisuke Yanagisawa, Shunta Komine, Rikuto Kubota, Masahito Ohue and Yutaka Akiyama, *Optimization of memory use of fragment extension-based protein-ligand docking with an original fast minimum cost flow algorithm*
* Shunsuke Teraguchi and Yutaro Kumagai, *Estimation of Diffusion Constants from Single Molecular Measurement without Explicit Tracking*
* Ernst Houtgast, Vlad‐Mihai Sima, Koen Bertels and Zaid Al‐Ars, *Hardware Acceleration of BWA-MEM Genomic Short Read Mapping with Longer Read Length*
* Iana Pyrogova and Limsoon Wong, *Protein complex prediction by date hub removal*
* Bobby Ranjan, Ket Hing Chong and Jie Zheng, *Composite mathematical modeling of calcium signaling behind neuronal cell death in Alzheimer's disease*
* Wen‐Yu Kuo, Shu‐Hwa Chen, Sheng‐Yao Su, Wei‐ChunChung, Jen‐Ming Ho, Henry Horng‐Shing Lu and Chung‐Yen Lin, *A Gene Profiling Deconvolution Approach to Estimating Immune Cell Composition from Complex Tissues*
* Aparajita Dutta, Tushar Dubey, Kusum Kumari Singh and Ashish Anand, *SpliceVec: distributed feature representations for splice junction prediction*
* Yu‐Chen Liu, Hsiao‐Chin Hong, Chi‐Dung Yang, Wei‐Hsiang Lee, Hsin‐Tzu Huang and Hsien‐Da Huang, *Ouroboros Resembling Competitive Endoge-nous Loop (ORCEL) in circular RNAs revealed through transcriptome sequencing dataset analysis*
* Shunyun Yang, Quan Zou, Shaoliang Peng and Runxin Guo, *cmFSM: A Scalable CPU-MIC Coordinated Drug-Finding Tool by Frequent Subgraph Mining*
* Yoichi Takenaka, Kazuma Mikami, Shigeto Seno and Hideo Matsuda, *Automated Transition Analysis of Activated Gene Regulation during Diauxic Nutrient Shift in Escherichia Coli and Adipocyte Differentiation in Mouse Cells*
* Wei‐Hsiang Kung, Chi‐Lung Lee, Chi‐Dung Yang, Ching‐
Fang Yu, Men‐Yee Chiew, Fang‐Hsin Chen and Hsien‐Da Huang, *Integrated microRNA and mRNA expression profile analysis of Tumor-Associated Macrophages after Exposure to Single-Dose Irradiation*
* Mengxuan Lyu, Jiaxing Chen, Yiqi Jiang and Shuaicheng Li, *KDiamend: a package for detecting key drivers in a molecular ecological network of disease*
* Guillaume Fertin, Falk Hüffner, Christian Komusiewicz and Manuel Sorge, *Matching Algorithms for Assigning Orthologs after Duplication Events*
* Weiping Sun, Yi Liu and Kaizhong Zhang, *An Approach for N-linked Glycan Identification from MS/MS Spectra by Target-Decoy Strategy*
* Kingshuk Mukherjee, Md Mahmudul Hasan, Christina Boucher and Tamer Kahveci, *Counting motifs in dynamic networks*
* Po‐Jung Huang, Chi‐Ching Lee, Ling‐Ya Chiu, Kuo‐Yang Huang, Yuan‐Ming Yeh, Chia‐Yu Yang, Cheng‐Hsun Chiu and Petrus Tang, *VAReporter: Variant reporter for cancer research of massive parallel sequencing*
* Y‐H. Taguchi, *Tensor decomposition/principal component analysis based unsupervised feature extraction applied to brain gene expression and methylation profiles of social insects with multiple castes*
* Petronela Buiga, Ari Elson, Lydia Tabernero and Jean‐Marc Schwartz, *Regulation of dual specificity phosphatases in breast cancer during initial treatment with Herceptin: A Boolean model analysis*
* Takuya Mori, Hayliang Ngouv, Morihiro Hayashida, Tatsuya Akutsu and Jose Nacher, *ncRNA-disease association prediction based on sequence information and tripartite network*
* Soorin Yim, Hasun Yu, Dongjin Jang and Doheon Lee, *Annotating activation/inhibition relationships to protein-protein interactions using Gene Ontology relations*
* Shuqin Zhang, *Comparisons of gene coexpression network modules in breast cancer and ovarian cancer*
* Anna Górska, Silke Peter, Matthias Willmann, Ingo Autenrieth, Robert Schlaberg and Daniel Huson, *Dynamics of the human gut phageome during the antibiotics treatment*
