---
layout: default
---

# The List of Accepted Posters

* [A1] __*Constrained Maximum Entropy Models Selecting Genotype Interactions for Interval Censored Disease Occurrence*__, Qing Pan and Aotian Yang
* [A2] __*Metabolite-content-guided prediction of edible/medicinal properties in plant bioprospecting*__, Liu Kang, Morita Aki, Kanaya Shigehiko and Altaf-Ul-Amin Md.
* [A3] __*Extraction of genotype- phenotype relationships for a specific disease domain using the text mining technique*__, Kiyong Kim, Jae Moon Shin, Eunsung Park, Jeehee Yoon and Keonbae Lee
* [A4] __*Prediction of GTP binding sites in Rab proteins based on two- dimensional convolutional neural networks and position specific scoring matrices profiles*__, Nguyen Quoc Khanh Le, Quang Thai Ho and Yu-Yen Ou
* [A5] __*Application of Multimodal Machine Learning Methods for Breast Cancer Detection*__, Dawn Chen and Ming-Jing Hwang
* [A6] __*ASSA-PBN: A Toolbox for Probabilistic Boolean Networks*__, Andrzej Mizera, Jun Pang and Qixia Yuan
* [A7] __*A simulation tool for generating DNA and methylation sequencing data*__, Chen-Yu Kang and Ren-Hua Chung
* [A8] __*Dissecting genomic determinants of plant epigenomes with deep convolutional neural networ*__, Ngoc Tu Le and Tu Bao Ho
* [A9] __*Missing value imputation for gene expression data with Deep Learning*__, Kazumasa Saito, Shigeto Seno, Hideo Matsuda and Yoichi Takenaka
* [A10] __*CAFE MOCHA: An Integrated Platform for Discovering Clinically Relevant Molecular Changes in Cancer; an Example of Distant Metastasis and Recurrence-linked Classifiers in HNSCC*__, Neeraja Krishnan, Mohanraj I, Janani Hariharan and Binay Panda
* [A11] __*Combining probabilistic alignments with read pair information improves accuracy of split-alignments*__, Naruki Yoshikawa, Anish Shrestha and Kiyoshi Asai
* [A12] __*Combined Knowledge-Free and Data-Driven Strategy for Ab Initio Identification of Transcription Factor Binding Motifs*__, Takuya Sumi, Asuka Sugimoto, Jiyoung Kang and Masaru Tateno
* [A13] __*Characterizing the 3-node and 4-node motifs in both cancer networks and signal transduction networks*__, Ka-Lok Ng, Chien-Hung Huang and Ci-Jun Peng
* [A14] __*Novel algorithm for Multicomponent Decomposition of Multivariate Data Coupled with Simulated Annealing-Extended Sampling*__, Masaru Tateno, Jiyoung Kang and KazuhikoYamazaki
* [A15] __*Novel algorithm for identification of operon-like gene clusters in eukaryotic whole genome DNA sequences*__, Kakeru Sakabe, Jiyoung Kang, Asuka Sugimoto and Masaru Tateno
* [A16] __*Clarification of human gut microbial community using Latent Dirichlet Allocation*__, Shion Hosoda, Suguru Nishijima, Tsukasa Fukunaga, Masahira Hattori and Michiaki
Hamada
* [A17] __*Fully automatic and intelligent adapter detection and trimming algorithms for single- and paired-end NGS reads*__, Chia-Hua Chang, Wen-Fu Hsieh and Jui-Hung Hung
* [A18] __*Functional analysis of metagenome using composition- based method*__, Meya Cho, Shigeto Seno, Hideo Matsuda and Yoichi Takenaka
* [A19] __*Detecting gene-gene interactions for mutivariate quantitative traits*__, Hyein Kim, Taesung Park and Mira Park
* [A20] __*Development of Clustering method using Factor Analysis based on Gene Expression of Functional Sets*__, Naoaki Ono, Nicolas Savaro, Ming Huang, Md Altaf-Ul-Amin and Shigehiko Kanaya
* [A21] __*A SOA-Based Library for Representation of the Biological Systems in Computer Algorithms*__, Sepehr Najjarpour and Mehrshad Khosraviani 
* [A22] __*Predicting Compound-Protein Interaction with Neural Networks based Pairwise Model*__, Mun-Hwan Lee, Eung-Hee Kim, Jinhyun Ahn, Jinuk Jung, Junho Park and Hong-Gee Kim 
* [A23] __*Learning mutation signatures of indels by Latent Dirichlet Allocation with Variational Bayes inference*__, Taro Matsutani 
* [A24] __*Multiple hybrid de novo genome assembly of an allotetraploid crop, finger millet, and homeolog phasing with one parental genome*__, Masaomi Hatakeyama, Sirisha Aluri, Mathi Thumilan Balachadran, Sajeevan Radha Sivarajan, Andrea Patrignani, Simon Grüter, Lucy Poveda, Rie Shimizu-Inatsugi, John Baeten, Kees-Jan Francoijs, Karaba N. Nataraja, Yellodu A. Nanja Reddy, Shamprasad Phadnis, Ramapura L. Ravikumar, Ralph Schlapbach, Sheshshayee M. Sreeman and Kentaro K. Shimizu 
* [A25] __*Accounting for Tumor Heterogeneity in Identification of Driver Patterns in Cancer Genomes*__, Weiqing Huang, Jie Hu and Lin Hou
* [A26] __*Ant colony optimization algorithm for predicting RNA folding pathways*__, Seira Takitou and Akito Taneda
* [B1] __*Chemosensitivity prediction using a combined gene expression profile in breast cancer*__, Ki-Yeol Kim
* [B2] __*Incorporating genetic networks into case-control association studies with high-dimensional DNA methylation data*__, Kipoong Kim and Hokeun Sun
* [B3] __*modPhEA: model organism Phenotype Enrichment Analysis of eukaryotic gene sets*__, Meng-Pin Weng and Ben-Yang Liao
* [B4] __*SCAST:A cost effective diagnostic framework based on cancer associated significant transcript screening*__, Anju Pratap and Michiaki Hamada
* [B5] __*Systematic Drug Repurpsing for Sepsis using public biomedical databases*__, Hyun Wook Han
* [B6] __*Design a pipeline for analyzing the mRNA degradation patterns*__, Jungim Won, Jaemoon Shin, Jeehee Yoon and Donghoon Jeong
* [B7] __*EXPath 2.0: an update version of database for identifying regulatory interactions in plant biological pathways*__, Guan-Zhen Li, Yu-Cheng Hung, Kuan-Chieh Tseng and Wen-Chi Chang
* [B8] __*Analyzing cell type-specific gene regulatory networks using Immuno-Navigator*__, Alexis Vandenbon
* [B9] __*Comparing ensembles of protein structures from molecular dynamics simulations in terms of residue-residue interaction*__, Chie Motono and Takatsugu Hirokawa
* [B10] __*A Design of DrugBank to BioPAX Converter*__, Jinhyun Ahn, Eung-Hee Kim, Munhwan Lee, Jinuk Jung, Junho Park and Hong-Gee Kim
* [B11] __*Plasma microRNA biomarker detection and evaluation for mild cognitive impairment using differential correlation analysis and logistic regression*__, Mitsunori Kayano, Sayuri Higaki and Shumpei Niida
* [B12] __*In Search of Structural Homologs in the Twilight Zone*__, Tsun-Tsao Huang and Jen-Shiang Yu
* [B13] __*PhenoToGeno Browser (PGB): an interactive genome browser to visualize and analyze disease- gene-variant relationships*__, Jae Moon Shin, Dawoon Jung and Jeehee Yoon
* [B14] __*The systematic model of drug repositioning considering disease type and epigenetic features*__, Kyungsik Ha, Junho Park, Hyunwhan Joe, Hwajin Lee and Hong-Gee Kim
* [B15] __*Genome-wide identification of graft-transmissible small RNAs using heterografts between Solanaceous species*__, Bo-Han Hou, Ruei-Yuan Su, Wen-Chi Lee, Shu-I Lin and Ho-Ming Chen
* [B16] __*Biological Interaction Data Model Development for Heterogeneous Data Integration*__, Junho Park and Hong-Gee Kim
* [B17] __*Development of User- Configurable NGS Pipeline System*__, Sung Min Yang, Yong Sun Shim, Hye Yeon Kim and Hong-Gee Kim
* [B18] __*Identification of Adverse Drug Reactions and Medication Intakes on Twitter Using Various Combinations of Language Features*__, Chen-Kai Wang, Emily Chia-Yu Su and Hong-Jie Dai
* [B19] __*CMV: Cancer Mutation Visulizer, a cross-cohort annotating and visualizing system for cancer analysis projects*__, Chi-Ching Lee and Po-Jung Huang
* [B20] __*Virtual screening of novel serine racemase inhibitors in silico using the molecular docking method*__, Jia-Jun Liu, Ming-Hsun Tsai, Dong-Chuan Wu, Wei-Yong Lin, Chung-Y. Hsu and Chih-Hao Lu
* [B21] __*Annie Yim, Prasanna Koti, Milena Duerrbaum,Salma Gamal, Cecilia Garcia-Perez, Fabiana
Perocchi, Zuzana Storchova and Bianca Habermann*__, Defective Electron Transport Chain Complex in Aneuploidy Cells - Exploring Mitochondrial Functions with RNA-seq Data Using the MitoXplorer web- platform
* [B22] __*The study of the sequence and structure similarity among metal ion binding sites in proteins*__, Ming-Hsun Tsai, Jia-Jun Liu and Chih-Hao Lu
* [B23] __*Optimizing sequence assembly for evolutionary variant pattern analysis*__, Seokwoo Kang, Hongtaek Oh and Giltae Song 
* [B24] __*Interpretation of Branchpoint Variations in Next Generation Sequencing*__, Poulami Chaudhuri, Akriti Jain and Rajgopal Srinivasan
* [B25] __*transcriptional networks in triple-negative breast cancer (TNBC)*__, Men-Yee Chiew, Chi-Dung Yang, Yu-Jen Chang and Hsien-Da Huang
* [B26] __*Development of a Multi-gene Sequencing Panel for Sudden Cardiac Death Screening Based on Preventive Medicine*__, Hsin-Tzu Huang, Wei-Chih Huang, Wei-Chi Wang, Yu-Feng Hu and Hsien-Da Huang
* [B27] __*Integrated analysis of small RNA sequencing for human cancer research*__, Shu-Hsuan Liu and Wei-Chung Cheng
* [B28] __*Improved member genome recovery from enrichment reactor microbial communities using long read metagenomics*__, Krithika Arumugam, Irina Bessarab, Xianghui Liu, Gayathri Natarajan, Cheryl Sze Yee Tsan, Stefan Wuertz, Federico Lauro, Ying Yu Law, Daniel Huson and Rohan Williams
* [B29] __*Feature extraction using wavelet transformation to reveal genomic variant effect on binding affinity of EGFR*__, Mayumi Kamada and Mikito Toda
* [B30] __*Fucosylation pathway genes are associated with patients survival late in early-stage non-small cell lung cancer*__, Jin-Muk Lim, Yong-Sun Shim, Jiwon Son and Hong-Gee Kim
* [B31] __*Erythrocytes simulation including detailed oxidative stress pathway for study of premature infants*__, Ayaka Matsuo and Yasuhiro Naito
* [B32] __*Links between Chromosomal Rearrangements and Chromatin Structure*__, Krister Swenson, Sylvain Pulicani, Pijus Simonaitis and Mathieu Blanchette
* [B33] __*Cosearge: an exploratory approach reveals spatial gene co- localization beyond topologically associated domains*__, Yutaka Saito and Toutai Mituyama 
* [B34] __*Optimization of biomarker quantification in salivary metaboloics*__, Atsumi Tomita, Masayo Mori and Masahiro Sugimoto
* [B35] __*Subcellular localization of lncRNAs is affected by Transposable Element (TE) derived sequences inside lncRNAs.*__, Takafumi Chishima and Michiaki Hamada 
* [B36] __*Genome Analysis of Five Ralstonia solanacearum strains from the Philippines*__, Ma. Anita Bautista, Jo-Hannah Llames, Lucille Villegas and Emilia Andrea Sabban
* [B37] __*Metagenomic Analysis of Stingless Bee Gut Microbiome*__, Alfredo Jose Ballesteros, Maria Dulce Mostoles, Ma. Carmina Manuel and Ma. Anita Bautista
* [B38] __*Whole-genome sequencing analysis of Esophageal Squamous Cell Carcinoma with response to neo-adjuvant chemotherapy*__, Munmee Dutta, Masashi Fujita, Tadashi Yasuda, Raúl Nicolás Mateos, Ashwini Patil, Kenta Nakai and Hidewaki Nakagawa 
* [C1] __*Structural insights into the polypharmacological activity of dietary flavonols on serine/threonine kinases*__, Bincy Baby, Priya Antony, Walaa Al Halabi, Zahrah Al Homedi and Ranjit Vijayan
* [C2] __*Classification of species by bi- clustering based on emitting volatile organic compounds*__, Mohammad Bozlul Karim, Naoaki Ono, Md.Altaf-Ul -Amin and Shigehiko Kanaya
* [C3] __*Compuational analysis of the mode-of-action and pharmacological effects of Japanese Kampo medicines*__, Ryusuke Sawada, Michio Iwata, Masahiro Umezaki, Yoshihiko Usui, Takaki Kubono, Makoto Kadowaki and Yoshihiro Yamanishi
* [C4] __*Generation of hypothetical reactions from a set of metabolites based on stoichiometry*__, Jun Ohta
* [C5] __*Toward efficient protein-ligand docking for virtual screening by reuse of fragments*__, Rikuto Kubota, Keisuke Yanagisawa, Masahito Ohue and Yutaka Akiyama 
* [C6] __*Time depndent variance- covariance matrix and random matrix analysis of protein-ligand dynamical interaction*__, Masanori Yamanaka 
* [C7] __*Feature extraction by enumerating lasso solutions for predicting plasma protein binding of peptide*__, Takashi Tajimi, Naoki Wakui, Masahito Ohue and Yutaka Akiyama 
